dotCMS/src/test/java/com/dotcms/util/pagination/TemplatePaginatorTest.java

```diff
@@ -0,0 +1,121 @@
+package com.dotcms.util.pagination;
+
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.templates.business.TemplateAPI;
+import com.dotmarketing.portlets.templates.model.Template;
+import com.dotmarketing.util.PaginatedArrayList;
+import com.dotmarketing.util.UUIDUtil;
+import com.liferay.portal.model.User;
+import org.junit.Test;
+
+import java.util.Date;
+import java.util.Map;
+
+import static com.dotcms.util.CollectionsUtils.map;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * test {@link TemplatePaginator}
+ */
+public class TemplatePaginatorTest {
+
+    @Test
+    public void testGetEmptyTemplateList() throws DotDataException, DotSecurityException {
+
+        final int totalRecords = 10;
+        final User user = mock(User.class);
+        final String filter = "filter";
+        final Map<String, Object> params = map("title", filter);
```

Comment:
Codacy found an issue: [The String literal "title" appears 4 times in this file; the first occurrence is on line 32](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6042990)